docker-compose.yml

```diff
@@ -25,6 +26,7 @@ services:
       retries: 5
 
   redis:
+    container_name: redi
```

Comment:
spelling mistake - 's' is missing in 'redi'

Reply:
😱 
```suggestion
    container_name: redis
```